datamade/la-metro-councilmatic
# Update Board Member Najarian's headshot and About page wording

## Overview

Updates board member Aja Najarian's headshot and wording on the About page referencing the Governor's representative on the Board of Directors.

### Checklist



## Testing Instructions

 * Verify that the board member's headshot is updated
 * Verify that the About page references the Governor's rep as 'non-voting' instead of 'ex-officio'

Handles #832 #833
